src/notification/actionable-notification.component.ts

```diff
@@ -92,10 +92,14 @@ export class ActionableNotification extends BaseNotification {
 	@HostBinding("class.cds--actionable-notification--toast") get toastVariant() { return this.notificationObj.variant === "toast"; }
 	@HostBinding("class.cds--actionable-notification--error") get isError() { return this.notificationObj.type === "error"; }
 	@HostBinding("class.cds--actionable-notification--info") get isInfo() { return this.notificationObj.type === "info"; }
-	@HostBinding("class.cds--actionable-notification--info-square") get isInfoSquare() { return this.notificationObj.type === "info-square"; }
+	@HostBinding("class.cds--actionable-notification--info-square") get isInfoSquare() {
+		return this.notificationObj.type === "info-square";
+	}
```

Comment:
I'm assuming we are using predefined line length since I don't see the rule for it, might be good to override to what it was before? 🤔